src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -441,6 +496,11 @@ class DydxChainTransactions implements AbacusDYDXChainTransactionsProtocol {
           const parseDelegations = this.parseToPrimitives(delegations);
           callback(JSON.stringify(parseDelegations));
           break;
+        case QueryType.GetNobleBalance:
+          const nobleBalance = await this.nobleClient?.getAccountBalance('uusdc');
```

Comment:
umm i thought `uusdc` is only used in gas -- does the ibc/.. denom also work?

Reply:
This is for the noble chain not our chain, which only uses `uusdc` for denom for testnet and mainnet. we have it hardcoded in v4-client and abacus as well.